compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Serializability.hs

```diff
@@ -193,6 +194,13 @@ checkInterface _mod0 iface = do
       checkType SRChoiceArg (snd (chcArgBinder ch))
       checkType SRChoiceRes (chcReturnType ch)
 
+  case NM.lookup (MethodName "_view") (intMethods iface) of
+    Nothing -> throwWithContext $ ENoViewFound (intName iface)
+    Just method ->
+      let err = EViewNotSerializable (intName iface) (ifmType method)
+      in
+      catchAndRethrow (const err) $ checkType SRDataType $ ifmType method
```

Comment:
While it's convenient to check both existence and serializability here, the existence check feels a bit out of place in this module. I think it would make more sense as part of [`DA.Daml.LF.TypeChecker.Check.checkIface`](https://github.com/digital-asset/daml/blob/main/compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Check.hs#L859)